datamonster_api/lib/datasource.py

```diff
@@ -4,18 +4,19 @@
 
 
 class Datasource(BaseClass):
-    """Representation of a data source in DataMonster
+    """Representation of a data source in Data Monster
 
-    :param _id: (str) uuid of the data source
-    :param name: (dict) name of the data source
+    :param _id: (str) uniquer internal identifier for the data source
+    :param name: (dict) name of the data source, including the vendor for the data
     :param category: (list) associated categories
-    :param uri: (str) DataMosnter uri associated with the data source
-    :param dm: DataMonster object
+    :param uri: (str) Data Monster resource identifier associated with the data source
+    :param dm: ``DataMonster`` object
 
     *property* **name**
-        **Returns** (str) datasource name
+        **Returns** (str) name of data source, including vendor
     *property* **category**
-        **Returns** (str) category associated with the datasource
+        **Returns** (str) category associated with the data source, e.g.,
+        Web Scrape Data or Uploaded Data
```

Comment:
the categories should have quotes or back ticks to make it clear what is the category (the or is not part of the category)